### DISCUSSION TITLE
how to get remote update to web app. - onConfigUpdated method for web

### DISCUSSION THREAD

**Discussion by Pragna-Bhatt** [firebase/flutterfire#13265] [Q&A]
Hi there
As a flutter dev; in my project I need to integrate flutter remote configuration.
I am able to fetch updated key value for IOS and android but not for web version.

I tried to fetch key value using below method

```
 _remoteConfig.onConfigUpdated.listen((event) {
      debugPrint('Config updated: ${event.updatedKeys}');
    });
```

But for web I am getting DartError: Unsupported operation: onConfigUpdated is not supported for web

Also, tried 
```
 Future<void> fetchAndActivate() async {
    bool updated = await _remoteConfig.fetchAndActivate();

    if (updated) {
      debugPrint('The config has been updated.');
    } else {
      debugPrint('The config is not updated.');
    }
  }
```
But it not auto triggers as an when remote changes made.

so, is there any way to fetch updates for web too?
Thanks in advance.

Note: 
I am using
firebase_remote_config: ^5.1.0

and flutter/dart version are:
Flutter 3.19.0 • channel stable 
Tools • Dart 3.3.0 • DevTools 2.31.1

**Comment by Juliusvm** [comment]
I have this same issue.
